program/src/utils.rs

```diff
@@ -181,5 +181,11 @@ pub fn get_existing_revision_map(
 /// revision map.
 pub fn get_latest_revision(rule_set_pda_info: &AccountInfo) -> Result<Option<usize>, ProgramError> {
     let (revision_map, _) = get_existing_revision_map(rule_set_pda_info)?;
-    Ok(revision_map.rule_set_revisions.last().copied())
+
+    match revision_map.rule_set_revisions.len() {
+        // we should always have at least one revision
+        0 => Err(RuleSetError::RuleSetRevNotAvailable.into()),
```

Comment:
It needs to be `RuleSetRevisionNotAvailable`, I will fix it

Reply:
Oh, did it just got renamed?

Reply:
yeah I did it in my test PR, sorry for inconvenience